groundstation/backend_api/passover.py

```diff
@@ -97,4 +98,36 @@ def post(self, local_data=None):
         }
         return response_object, 201
 
+    @create_context
+    def head(self, local_args=None):
+        """Temporary endpoint to just send slack message
+        
+        :param dict local_args: This should be used in place of the QUERY PARAMS that would be used through HTTP, used for local calls.
+
+        :returns: response_object, status_code
+        :rtype: tuple (dict, int)
+        """
+
+        current_time = datetime.datetime.now(datetime.timezone.utc)
+        passover = Passover.query.filter(Passover.timestamp < current_time).order_by(Passover.timestamp.desc()).limit(1).first()
```

Comment:
Maybe we should add a parameter to this endpoint that allows the caller to specify whether they want the timestamp from the most recent passover to be sent or the timestamp from the soonest future passover to be sent? What are your thoughts?